samples/bluetooth/fast_pair/locator_tag/src/main.c

```diff
@@ -58,6 +58,11 @@ static bool fmdn_provisioned;
 static bool fmdn_recovery_mode;
 static bool fmdn_id_mode;
 static bool fp_account_key_present;
+static bool fp_adv_ui_request;
+
+static const char *fp_adv_trigger_clock_sync = "clock_sync";
+static const char *fp_adv_trigger_ak_write = "ak_write";
```

Comment:
I would suggest changing name for this trigger

```suggestion
static const char *fp_adv_trigger_fmdn_provisioning = "fmdn_provisioning";
```